CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - Add support for auto-generated DSL and Job DSL extensions by executing the Job DSL scripts in a local Jenkins
   instance.
+- Upgrade job-dsl-core to 1.67.
```

Comment:
Please rebase and put this in the new section for version 3.1.0.

Reply:
Done.